tests/unit/components/SharedComponents/DateDisplay.test.js

```diff
@@ -1,26 +1,34 @@
 import { screen } from "@testing-library/react-native";
 import { DateDisplay } from "components/SharedComponents";
+import initI18next from "i18n/initI18next";
 import React from "react";
 
 import { renderComponent } from "../../../helpers/render";
 
 describe( "DateDisplay", () => {
+  beforeAll( async ( ) => {
+    await initI18next( );
+  } );
+
   it( "should be accessible", () => {
-    expect( <DateDisplay dateTime="2023-12-14T21:07:41-09:30" /> ).toBeAccessible( );
+    expect( <DateDisplay dateTime="2023-12-14T21:07:41+00:00" /> ).toBeAccessible( );
   } );
 
   it( "should format datetime correctly from date string", () => {
-    renderComponent( <DateDisplay dateTime="2023-12-14T21:07:41-09:30" /> );
-    expect( screen.getByText( "12/15/23 6:37 AM" ) ).toBeTruthy( );
+    renderComponent( <DateDisplay dateTime="2023-12-14T21:37:00+00:00" /> );
+    expect( screen.getByText( "12/14/23 9:37 PM" ) ).toBeTruthy( );
   } );
 
-  it( "should format datetime correctly from date string", () => {
-    renderComponent( <DateDisplay dateTime="2023-01-02T21:09:41-23:30" /> );
-    expect( screen.getByText( "1/3/23 8:39 PM" ) ).toBeTruthy( );
+  it( "should show the date in the local time zone by default", () => {
+    renderComponent( <DateDisplay dateTime="2023-01-02T08:00:00+01:00" /> );
+    expect( process.env.TZ ).toEqual( "UTC" );
```

Comment:
For this test to succeed, the process time zone must be UTC. I wrote this to test that expectation myself while writing this test, and I left it in because if that expectation is not met in the future, this will provide a clearer indication of why the test failed. So yes, it's to ensure a developer (or a CI env) has set things up correctly, but also to communicate to developers what wasn't set up correctly if/when it fails.